### ISSUE TITLE
Wikimedia Foundation websites bugs - part 3

### ISSUE THREAD

**Issue by Tiagoquix** [darkreader/darkreader#13151]

**Comment by Myshor** [comment]
First task for part 3 done.

**Comment by Myshor** [comment]
That will be easy. Wiki set nice class for the thing they should not invert 😁
`.mw-no-invert`

**Comment by Myshor** [comment]
I will add fixes in a while... BTW If you have possibility then re-open the issue. I have some holidays planned in near future and if it will be closed then you will wait until I will come back from holiday. 😉

**Comment by Myshor** [comment]
Everything mentioned after my holiday will be fixed in a minute. Hope it won't break older fixes as I needed to change or remove some older entries. 🫤

**Comment by Myshor** [comment]
Will add fix in a minute.

**Comment by Myshor** [comment]
I made some changes to selectors in new fix.
I'll push it in a minute.

**Comment by Myshor** [comment]
One missing bracket in my last changes messed that 😝
I will push the fix in a minute.

**Comment by Myshor** [comment]
Ok... First I fix the inverts of photos and colors in presidents.

As for that Tools button it's pain in the ass. Dropdown menu is always showing "under" the article and can not be clicked. 😒

**Comment by Myshor** [comment]
I'll leave it open for a while.

As I understand... making inverts and correcting colors is not the solution we want. We want the menu working, right?? 😝

**Comment by Myshor** [comment]
Ok... I found what caused this and have fix already. 😅

**Comment by Myshor** [comment]
The best solution for now should be:

- Remove whole wikipedia fixes. 😝
- As the button [Hide] works fine when I disabled all fixes for wikipedia locally.

What I found... if some button doesn't work and has INVERT applied it works when invert is removed.

I managed to fix that [Hide] button and all mentioned cdx-icons (I bet there will be more with your wikipedia browsing 😝)

As I wrote in the beginning.
The best solution would be to remove all fixes and start doing them from the scratch as without them wikipedia works fine but has some color fuckups. 😂
